nx/lib/nx.ex

```diff
@@ -7842,6 +7842,95 @@ defmodule Nx do
     impl!(tensor).sort(tensor, tensor, axis: axis, comparator: comparator)
   end
 
+  @doc """
+  Solve the equation a x = b for x, assuming a is a triangular matrix.
+
+  ## Options
+
+    * `:trans` - Type of system to solve, if none is provided defaults to: a x = b - Optional
+
+  | trans                    | system                        |
+  | ------------------------ | ------------------------------|
+  | `nil` or 0 or 'N'        | a x = b                       |
+
+    * `:lower` - Use only data contained in the lower triangle of a. Default is to use lower triangle, Optional
```

Comment:
I have removed the options for now since they always have no effect. We can add them back once implemented!

Reply:
@josevalim Understood!